src/server.rs

```diff
@@ -70,6 +70,13 @@ pub fn extract_cert_from_conn(connection: &Connection) -> Result<Vec<Certificate
 }
 
 pub fn certificate_info(cert_info: &[CertificateDer]) -> Result<(String, String)> {
+    certificate_info_default(cert_info, true)
+}
+
+pub fn certificate_info_default(
```

Comment:
I don't get what the postfix `_default` exactly mean. @kimhanbeom or @henry0715-dev, could you anyone tell me about it?

Reply:
@sehkone 
함수 명을 certificate_info_print_client_nm 이렇게 수정하면 조금더 직관적일까요?

Reply:
trait을 활용하여 변경하는 것도 고려해 보도록 하겠습니다.

Reply:
@henry0715-dev @kimhanbeom 

I suggest providing two public methods: `subject_from_cert` and `subject_from_cert_verbose`. Additionally, we need to define a private method, such as `subject_from_cert_opt`, as follows:

```Rust
pub fn subject_from_cert(cert_info: &[CertificateDer]) -> Result<(String, String)> {
    subject_from_cert_opt(cert_info, false)
}

pub fn subject_from_cert_verbose(cert_info: &[CertificateDer]) -> Result<(String, String)> {
    subject_from_cert_opt(cert_info, true)
}

fn subject_from_cert_opt(cert_info: &[CertificateDer], logging: bool) -> Result<(String, String)> {
     ...
}
```

The rationale behind this is that we can provide clearer names for users of the public methods, while the function that is difficult to name can remain private. BTW, the word default is hardly meaningful in this context.

Reply:
최초의 의도는 flag외에도 더 많은 파라미터들이 추가될 수도 있을것이란 가정으로 생각하고 간단하게 만들었습니다.
위에 언급해주신 것과 같은 형태로 만들게 된다면 파라미터 추가 되었을때, 영향이 있는 소스를 수정하는것은 불편함이 발생할 것이라 생각 됩니다.

Reply:
subject_from_cert_opt 메소드명은 좋은것 같습니다.
아래와 같이 2개의 함수로만 사용하는것은 어떨까요?

```
pub fn subject_from_cert_opt(cert_info: &[CertificateDer], logging: bool) -> Result<(String, String)>
pub fn subject_from_cert(cert_info: &[CertificateDer]) -> Result<(String, String)>
```

Reply:
@henry0715-dev 
While your suggestion is reasonable, it’s not certain that we will need additional options besides this logging in the future. Since it would be safe to follow your suggestion if more options become necessary, I believe the focus should be on providing public methods with clearer names for now. I think the prefix **opt** is too vague, so I believe it is more appropriate to use it in private methods, in this context.

Reply:
What I wanted to point out is that `certificate_info` only calls another public method, `certificate_info_default`. According to your suggestion, `subject_from_cert` would call `subject_from_cert_opt` in the same way, even though `subject_from_cert_opt` can do exactly what `subject_from_cert` does. In other words, `subject_from_cert_opt` provides two different features, but one method provides only one of these features, while no other method provides the other, which creates an imbalance. In this context, the two mutually exclusive features should be provided separately: one with logging and the other without logging. From the user's perspective, it would be easier to understand if the two different methods each provided only one of these two features.

Reply:
@sehkone 
새로운 이슈 생성해서 말씀주신 아래의 코드로 변경하도록 하겠습니다.
```
pub fn subject_from_cert(cert_info: &[CertificateDer]) -> Result<(String, String)> {
    subject_from_cert_opt(cert_info, false)
}

pub fn subject_from_cert_verbose(cert_info: &[CertificateDer]) -> Result<(String, String)> {
    subject_from_cert_opt(cert_info, true)
}

fn subject_from_cert_opt(cert_info: &[CertificateDer], logging: bool) -> Result<(String, String)> {
     ...
}
```